myuw/templates/base.html

```diff
@@ -114,14 +114,10 @@ <h1 class="sr-only">MyUW</h1>
 
         <!-- message banner (desktop only) -->
         <div class="myuw-banner-desktop" id="app_messages" role="complementary">
-
             <div class="message-banner">
-                {% if user.affiliations.grad %}
                 <div class="message-text">
-                    <h2>Welcome to MyUW!</h2>&nbsp; MyUW shows you personal, critical, and timely information. <a id="modal_toggle" href="#" class="btn btn-link" data-toggle="modal" data-target="#message_modal">Learn how it works.</a>
+                    <h2>New in MyUW</i><i class="fa fa-bullhorn" aria-hidden="true" style="font-size: 20px; padding: 0 6px 0 6px;"></i>&nbsp;</h2>Quick Links, Pages, Search, and more! &nbsp;<a id="modal_toggle" href="#" class="btn btn-link" data-toggle="modal" data-target="#message_modal">Learn what's new in MyUW</a>
```

Comment:
Actually, looks like you have a random closing </i> tag in here you'll want to fix.

`<h2>New in MyUW</i>`